appcelerator/titanium_mobile
# [NATIVESDK] Updating Titanium Repo with the latest Version of APSAnalytics -0.8.1

**TESTING INSTRUCTION**

*Create any app with appc studio. 
*run app on device . 
*log onto dashboard. 
*click on analytics tab and make sure analytics (active session shows up on dashboard)
